bin/gobblin-admin.sh

```diff
@@ -1,142 +0,0 @@
-#!/bin/bash
```

Comment:
How about maintaining compatibility with callers of the old script? One option is to leave the old scripts as wrapper scripts that call the new one. Another option is to have the old script names be symlinks to the new script and detect the usage based on the value of $0.

Reply:
I was targeting to cleanup and simplify those too many scripts Gobblin has, which could be confusing for new users. user should really move to new scripts, so not sure how much the backward compatibility is required for scripts, as oppose to APIs, otherwise we will never be able to clean up. but I see ur point so may be I will create wrapper as you suggested with note to remove that in future.